lfucg-exactions/client/js/components/LotForm.js

```diff
@@ -233,22 +233,28 @@ class LotForm extends React.Component {
                                                 </div>
                                                 <div className="col-sm-6">
                                                     <FormGroup label="Permit ID" id="permit_id">
-                                                        <input type="text" className="form-control" placeholder="Permit ID" disabled={currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0} />
+                                                        <input type="text" className="form-control" placeholder="Permit ID" />
                                                     </FormGroup>
-                                                    {currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0 &&
-                                                        <span className="help-block">This lot still has {currentLot.lot_exactions.current_exactions} in exactions. You may add a Permit ID when exactions are paid.</span>
-                                                    }
                                                 </div>
                                             </div>
                                         </fieldset>
-                                        <button disabled={!submitEnabled} className="btn btn-lex">Submit</button>
-                                        {!submitEnabled ? (
-                                            <div>
-                                                <div className="clearfix" />
-                                                <span> * All required fields must be filled.</span>
+                                        <div className="row">
+                                            <div className="col-sm-6">
+                                                <button disabled={!submitEnabled} className="btn btn-lex">Submit</button>
+                                                {!submitEnabled ? (
+                                                    <div>
+                                                        <div className="clearfix" />
+                                                        <span> * All required fields must be filled.</span>
+                                                    </div>
+                                                ) : null
+                                                }
+                                            </div>
+                                            <div className="col-sm-6">
+                                                {currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0 &&
```

Comment:
This red block could be overlooked.  Can we use onBlur and either open a modal or populate the same message in a line by itself in larger font?